src/Pages/Reviews.jsx

```diff
@@ -2,25 +2,32 @@ import { useEffect, useState } from "react";
 import { getReviews } from "../utils/utils";
 import ReviewCard from "../components/ReviewCard";
 import { Link } from "react-router-dom";
+import SortDropdown from "../components/SortDropdown";
 
 function Reviews() {
-    const [reviews, setReviews] = useState('')
-    const [isLoading, setIsLoading]= useState(true)
-    useEffect(()=>{ getReviews()
-        .then((rev) => {
-            setReviews(rev)
-            setIsLoading(false)
-        })
-    }, [])
-    if (isLoading) {
-       return <h2>loading</h2>
-    }
-    return (
+  const [reviews, setReviews] = useState('')
+  const [isLoading, setIsLoading] = useState(true)
+  const [sort, setSort] = useState("created_at");
+  const [order, setOrder] = useState('desc')
+  useEffect(()=>{ getReviews(sort,order)
+    .then((rev) => {
+        setReviews(rev)
+        setIsLoading(false)
+       })
+    }, [sort, order])
+  if (isLoading) {
+    return <h2>loading</h2>
+  }
+  return (
+    <div>
+      <h2>Reviews</h2>
+      <SortDropdown setSort={setSort} order={order} setOrder={setOrder} />
         <ul>
```

Comment:
Indentation is wrong for this section. Consider using ES lint to spot issues throughout your project